README.md

```diff
@@ -421,6 +423,9 @@ following properties:
 
     * `reason` (String) -- A string describing the reason for an unsupported video recommendation.
       For example, `'No camera was found.'`
+    
+    * `qualityLimitationReason` (String) --  The qualityLimitationReason indicates the reason behind 
+       any limitations in media quality during a session. It can have values: `cpu` for CPU overload, `bandwidth` for insufficient network bandwidth and `none` if there is no limition.
```

Comment:
Is this accurate:

```suggestion
    * `qualityLimitationReason` (String) --  Indicates the reason behind
       the highest resolution tested failing. It can have values: `'cpu'` for CPU overload, `'bandwidth'` for insufficient network bandwidth, and `'none'` if there is no limitation.
```